pkg/apiserver/apiserver.go

```diff
@@ -27,8 +27,11 @@ import (
 	"path"
 	rt "runtime"
 	"strconv"
+	"strings"
 	"time"
 
+	"k8s.io/client-go/pkg/util/sets"
+	"k8s.io/client-go/pkg/version"
```

Comment:
It's unfortunate goimports finds common types in the client-go pkg first. I think we want our own sets type